src/uwtools/tests/utils/test_templater.py

```diff
@@ -1,21 +1,84 @@
-# pylint: disable=missing-function-docstring
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
 """
 Tests for uwtools.utils.templater module.
 """
 
+import yaml
+from pytest import fixture, raises
+
+from uwtools.logger import Logger
 from uwtools.utils import templater
 
 
-def test_render():
-    # WRITE TEST(S) TO EXPLICITLY TEST THIS UNIT.
-    pass
+@fixture
+def config_file(tmp_path):
+    path = tmp_path / "config.yaml"
+    yaml = """
+roses: red
+violets: blue
+cannot:
+    override: this
```

Comment:
Hmm interesting - I hadn't really tested them at deeper key levels so this may be an artifact of earlier and simpler implementations. I think the reference configs we had been looking at were also limited to 1-2 levels.